business/checkers/authorization_policies_checker.go

```diff
@@ -59,7 +59,7 @@ func (a AuthorizationPolicyChecker) runChecks(authPolicy security_v1beta.Authori
 		common.SelectorNoWorkloadFoundChecker(AuthorizationPolicyCheckerType, matchLabels, a.WorkloadList),
 		authorization.NamespaceMethodChecker{AuthorizationPolicy: authPolicy, Namespaces: a.Namespaces.GetNames()},
 		authorization.NoHostChecker{AuthorizationPolicy: authPolicy, Namespace: a.Namespace, Namespaces: a.Namespaces,
-			ServiceEntries: serviceHosts, ServiceList: a.ServiceList, VirtualServices: a.VirtualServices, RegistryServices: a.RegistryServices},
+			ServiceEntries: serviceHosts, VirtualServices: a.VirtualServices, RegistryServices: a.RegistryServices},
```

Comment:
Mentioned before, the "ServiceEntries" information should be included using the "RegistryServices" data.
![image](https://user-images.githubusercontent.com/1662329/148771880-9cd30f52-987c-4f11-9ded-7458c5b92cee.png)

I think it's worth simplifying the logic, then the checkers may contain fewer data to iterate, and overall it could be a good move.

Reply:
Agree, added a comment on https://github.com/kiali/kiali/issues/4528 to have this done in next iteration.

Reply:
@lucasponce back to this topic, actually ServiceEntries are listed in a exported Istio configs retrieved from Registry.
do we need to move ServiceEntries into RegistryServices? because RegistryServices is an array of kubernetes.RegistryService which contains IstioService struct, and it ServiceEntry will be odd within this struct, wdyt?

in my mind was to unify ServiceEntries into one array listed from Registry and pass it to all validations.